components/provisioner/internal/operations/executor.go

```diff
@@ -115,6 +115,10 @@ func (e *Executor) process(operation model.Operation, cluster model.Cluster, log
 
 		result, err := step.Run(cluster, operation, log)
 		if err != nil {
+			if errors.Is(err, errors.New("cluster kubeconfig is nil")) {
+				log.Warnf("Warning, the %s", err.Error())
```

Comment:
```suggestion
				log.Warnf("Warning, the %s", err)
```

Calling `.Error()` here is pointless

Reply:
it's now still err.Error()..
btw, may I ask why `break` when `errors.Is(err, ErrKubeconfigNil)` in executor? After break, it will go to `e.updateOperationStatus(logger, operation.ID, "Operation succeeded", model.Succeeded, time.Now())`, then if `ErrKubeconfigNil`, the provision operation will be considered as succeeded?

Reply:
it's now still err.Error()..
btw, may I ask why `break` when `errors.Is(err, ErrKubeconfigNil)` in executor? After break, it will go to `e.updateOperationStatus(logger, operation.ID, "Operation succeeded", model.Succeeded, time.Now())`, then if `ErrKubeconfigNil`, the provision operation will be considered as succeeded?